.github/scripts/ci_variables.py

```diff
@@ -5,9 +5,9 @@
 
 # CI instance environment variables
 # This is used as a unique tag for all instances launched in a workflow
-ci_workflow_id = os.environ['CIRCLE_WORKFLOW_ID']
-ci_commit_sha1 = os.environ['CIRCLE_SHA1']
+ci_workflow_id = os.environ['GITHUB_RUN_ID']
```

Comment:
You should rename instances of workflow with "run" or w.e the GH-A equivalent is.